### ISSUE TITLE
Rocking curve theta and energy range converter

### ISSUE THREAD

**Issue by saga111a** [kklmn/xrt#179]
I dont find this issue, but may some one post about it
When use XRT Crystall reflectivy calculator plot with scan units as eV should be inverse. i.e. plot with $\theta$ units shoud be "mirrored" when convert to eV units. 
2d sin($\theta$) = hc / E i.e. E depends inversely on $\theta$
Energy units
![image](https://github.com/kklmn/xrt/assets/20823297/745d6357-da34-4882-8c46-76d7b838575d)
Angle units
![image](https://github.com/kklmn/xrt/assets/20823297/d635000a-591c-46de-802e-adc01e7b3394)

**Comment by kklmn** [comment]
Yes, indeed. Thank you for spotting it!
Will be fixed soon.

**Comment by kklmn** [comment]
I was also tricked by the fact that an increased Bragg angle corresponds to a decreased energy and so the reflectivity pictures should be mirrored when upon variable change E <-> θ. A better argument is this: the wave number of the formed standing wave is Q=2k sin(θ)=(E/hc)sin(θ). So both increase of E or increase of θ can change the condition for the standing wave being in phase or out of phase with the lattice: this is where we change from the crispy end of the Darwin table to the washed out end of it.

In conclusion, our curves look correct in all possible variables.

**Comment by saga111a** [comment]
Yes, you are right. Thank you! I realized that using the Bragg equation in this case is incorrect.
